utility/utils.py

```diff
@@ -1408,3 +1408,44 @@ def clone_the_repo(config, node, path_to_clone):
     log.info(f"repo_url: {repo_url}")
     git_clone_cmd = f"sudo git clone {repo_url} -b {branch}"
     node.exec_command(cmd=f"cd {path_to_clone} ; {git_clone_cmd}")
+
+
+def install_fio(client_node):
+    """Install fio package on client node.
+
+    Args:
+        client_node: ceph node
+    """
+    log.info("Installing fio on client node")
+    client_node.exec_command(cmd="yum install fio -y", sudo=True)
```

Comment:
@VasishtaShastry can we please make sure if fio package is available by default or anything else has to be done to get it installed.

Reply:
> @VasishtaShastry can we please make sure if fio package is available by default or anything else has to be done to get it installed.
From cephci perspective, we can install on every node (easily) or can explore to install just on clients which gets created at the time of prerequisites
Is that okay ? (Though its very less, there would be additional time involved when we observe from pipeline execution perspective)